systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterTest.java

```diff
@@ -152,4 +154,36 @@ String globalVariableJsonPathBuilder(String variable) {
                 .filter(event -> event.getInvolvedObject().getName().equals(resourceName))
                 .collect(Collectors.toList());
     }
+
+    public void sendMessages(List<String> messages, String clusterName, String topic) {
+        LOGGER.info("Sending messages");
+        String messagesToSend = "";
+        for (int i = 0; i < messages.size(); i++) {
+            messagesToSend = messagesToSend + messages.get(i) + "\\n";
+        }
+        messagesToSend = messagesToSend.substring(0, messagesToSend.length() - 2);
+
+        String command = "echo -e \"" + messagesToSend + "\" | sh bin/kafka-console-producer.sh --broker-list " +
+                clusterName + "-kafka:9092 --topic " + topic + " & sleep 20; kill %1";
+        try {
```

Comment:
What is a reason of using try/catch for this code? Can we remove it? or if we have some reason can we add logic to catch block?